mitodl/teachersportal
# Implemented shopping cart

Including:
- [x] Sidebar
- [x] Quantity Slider
- [x] Add selections to cart
- [x] Calculate total
- [x] Edit cart

To test manually:
- [Assumption] You have a course (an oscar product) and some modules (some variants of that product) in your local db. If not, follow instructions in: https://github.com/mitodl/teachersportal/pull/127
- Visit http://192.168.99.100:8075/courses/0ff00bdc-36d3-43ea-9359-6417182c44c2 (or whatever the uuid of the course you created was)
- Click the "Buy" tab on the course detail page.
- You should see a slider to be used to select the number of seats to be purchased. Slide it to select a number of seats.
- You should see a listing of course modules which can be purchased. Check the modules you wish to purchase
- Click "Update Cart"
- You should see a cart sidebar slide out with your selections listed, and a checkout button.
- Click "Checkout"
- In the checkout form, enter an address, click "payment info"
- Enter 4242-4242-4242-4242 as your cc# All other fields can be filled however you like.
- Click "Pay $X"
- You should get an email confirming your purchase.

fixes #122
